src/wasm-binary.h

```diff
@@ -1059,6 +1059,18 @@ enum MemoryAccess {
 
 enum MemoryFlags { HasMaximum = 1 << 0, IsShared = 1 << 1, Is64 = 1 << 2 };
 
+enum ElementFlags {
+  // Bit 0: 0 means active segment, 1 means passive or declarative
+  IsInactive = 1 << 0,
```

Comment:
I think it will be clearer to match the spec language here.

```suggestion
  IsPassive = 1 << 0,
```

Reply:
Bit zero means "it might be passive or declarative. look at bit 1"! That's why I went for `IsInactive` as `IsPassive` might have been confusing.

Reply:
Unless I'm reading this the wrong way and a "declarative" segment is also considered "passive"? I was just reading the related section in the Bulk Memory Operations [overview doc](https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md#element-segments) and I think you're right.

Reply:
Went with `IsPassive` as you suggested.